test/unit/gateways/quickpay_test.rb

```diff
@@ -176,7 +176,7 @@ def expected_store_parameters_v6
       "protocol"=>["6"],
       "msgtype"=>["subscribe"],
       "merchant"=>["LOGIN"],
-      "md5check"=>["8b7b3100fb835646723b9583febfb228"]
+      "md5check"=>["bd07f96fcb7e2288b88e7af603f1eb9a"]
```

Comment:
This test expectation actually figured out the year a few lines up, but then this hash gets changed anyway.